x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -16,25 +17,39 @@ import { useLocalStorage } from '../../../hooks/useLocalStorage';
 import { FETCH_STATUS, useFetcher } from '../../../hooks/use_fetcher';
 import { useUpgradeAssistantHref } from '../../shared/Links/kibana';
 import { SearchBar } from '../../shared/search_bar';
+import { getTimeRangeComparison } from '../../shared/time_comparison/get_time_range_comparison';
 import { NoServicesMessage } from './no_services_message';
 import { ServiceList } from './service_list';
 import { MLCallout } from './service_list/MLCallout';
 
 const initialData = {
-  items: [],
-  hasHistoricalData: true,
-  hasLegacyData: false,
+  requestId: '',
+  mainStatistics: { items: [], hasHistoricalData: true, hasLegacyData: false },
```

Comment:
Perhaps simply `response`. Also more in line with `requestId`

```suggestion
  response: { items: [], hasHistoricalData: true, hasLegacyData: false },
```